includes/models/model.llms.student.php

```diff
@@ -1435,6 +1462,70 @@ public function unenroll( $product_id, $trigger = 'any', $new_status = 'expired'
 
 	}
 
+	/**
+	 * Delete a student enrollment.
+	 *
+	 * @since [version]
+	 *
+	 * @param  int     $product_id  WP Post ID of the course or membership.
+	 * @param  string  $trigger     Optiona. Only delete the student's enrollment if the original enrollment trigger matches the submitted value
+	 *                              "any" will remove regardless of enrollment trigger. Default "any".
+	 * @return boolean Whether or not the enrollment records have been succesfully removed.
+	 *
+	 * @see  llms_delete_student_enrollment() calls this function without having to instantiate the LLMS_Student class first.
```

Comment:
Despite the pre-existing `@see` tags in other areas of this class this tag should appear below the changelog tags (`@since`) see doc: https://github.com/gocodebox/lifterlms/blob/master/docs/documentation-standards.md#functions-and-class-methods

Reply:
right, bad copy&paste :D 
fixed in 9bd326b